Doc/library/sqlite3.rst

```diff
@@ -21,6 +25,12 @@ The sqlite3 module was written by Gerhard Häring.  It provides an SQL interface
 compliant with the DB-API 2.0 specification described by :pep:`249`, and
 requires SQLite 3.7.15 or newer.
 
+
```

Comment:
Do you think it would be useful to have a ToC or even just a sentence with direct links to the main sections?

Something like:
```rst
If you are new to sqlite3, read the :ref:`sqlite3-tutorial`; for information about the objects
and their methods, skip to the :ref:`sqlite3-reference`; if you are trying to solve a particular
issue, check the :ref:`sqlite3-howtos`;  and for more theoretical background on the module,
see the :ref:`sqlites-concepts`.
```

or

```rst
This documented includes four main sections:
* :ref:`sqlite3-tutorial`: teaches how to use the sqlite3 module;
* :ref:`sqlite3-reference`: describes the classes and methods defined in the module;
* :ref:`sqlite3-howtos`: explains how to solve specific problems;
* :ref:`sqlites-concepts`: provides more background and insights about the design of the module;
```

(We might want to standardize this as we start adopting diataxis more widely.)

Reply:
Definitely; IMO the latter is the best option.